extensions/interactions/base_test.py

```diff
@@ -313,6 +337,28 @@ def test_default_interactions_are_valid(self):
             else:
                 self.assertIsNone(interaction.default_outcome_heading)
 
+            # Check that the rules for this interaction have object editor
+            # templates.
+            for rule_name, rule_dict in interaction.rules.iteritems():
+                rule_description = rule_dict['description']
+                param_list = rule_domain.get_param_list(rule_description)
+
+                for (_, param_obj_type) in param_list:
+                    # TODO(sll): Get rid of these special cases.
```

Comment:
Why do we need special cases? If any objects don't have object editor templates, can they be removed?